PendingReleaseNotes.md

```diff
@@ -15,6 +15,10 @@ v1.7...
   - See [the design](https://github.com/rook/rook/blob/master/design/ceph/resource-dependencies.md)
     for detailed information.
 
+- OSDs
+  - Only GPT partitions can be used for OSDs. Partitions without a type or with any other type will
```

Comment:
If we're backporting this, let's revert the pending release notes to avoid merge conflicts.

Reply:
I think I would prefer to keep the notes and just edit the backport, that way we don't forget.

Reply:
When the release notes are created for patch releases, the pendingReleaseNotes.md is not consulted. This is only for the minor releases, so if we need a special note in the release this won't help anyway.